cosmoz-image-viewer.js

```diff
@@ -229,20 +230,24 @@
 		previousImage() {
 			this.$.carousel.prev();
 		},
+
+		_createImageOverlay() {
+			let dialog = document.createElement('cosmoz-image-viewer-overlay');
+			dialog.id = 'cosmoz-image-viewer-overlay';
+			dialog.noCancelOnOutsideClick = true;
+			dialog.loop = this.loop;
+			dialog.showDetach = this.showDetach;
+			document.body.appendChild(dialog);
+			this.imageOverlay = imageOverlay = dialog;
```

Comment:
What is `this.imageOverlay` ? Is it needed somewhere else ?

Reply:
@plequang It's the shared dialog to avoid creating more than one. The old code:
``let dialog = document.querySelector('#cosmoz-image-viewer-overlay'); ``
didn't worked in Firefox because it's not using Polymer API to get the element.